src/style.css

```
@@ -0,0 +1,48 @@
+@font-face {
+  src: url(fonts/Avenir-Book.ttf);
+  font-family: 'Avenir';
+  font-weight: 400;
+}
+
+@font-face {
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-family: 'Avenir';
+  font-weight: 900;
+}
+
+.search__input {
+  display: block;
+  width: 50%;
+  height: 70px;
+
+  background: url(../src/images/Search.svg) no-repeat;
+  background-position: 25px 25px;
+  padding-left: 60px;
+  font-family: Avenir;
+  font-size: 16px;
+  font-weight: 400;
+
+  color: #3D4E61;
+
+  border: 1px solid #E1E7ED;
+  box-sizing: border-box;
+  box-shadow: 0px 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  box-shadow: 0px 1px 8px 0px #3D4E611A;
+}
+
+.search__input:hover {
+  box-shadow: 0px 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search__input:focus {
+  font-weight: 900;
+  background-color:  rgb(245,245,246);
```

Тут должен быть linear-gradient, а не просто серый цвет
```suggestion
  background-color: rgb(245,245,246);
```

---

Я когда задаю linear то пропадает иконка поиска 
почему, как исправить?